CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * [CHANGE] Ingester: Remove EnableNativeHistograms config flag and instead gate keep through new per-tenant limit at ingestion. #6718
 * [CHANGE] StoreGateway/Alertmanager: Add default 5s connection timeout on client. #6603
 * [CHANGE] Validate a tenantID when to use a single tenant resolver. #6727
+* [CHANGE] Ingester/StoreGateway: Change error code and configuration of resource-based limiter. #6771
```

Comment:
We can ignore the changelog as resource based limiter has not been released yet so it is not user facing change. Or you can add 6771 to the original changelog entry.